galette/templates/default/components/dynamic_fields.html.twig

```diff
@@ -75,7 +75,7 @@
                     {% endfor %}
                 </select>
             {% elseif get_class(field) == 'Galette\\DynamicFields\\Date' %}
-                {% if preferences.pref_field_information_position == 'above' %}
+                {% if field.hasInformationAbove() %}
                     <div class="exemple">{{ _T("(yyyy-mm-dd format)") }}</div>
                 {% endif %}
```

Comment:
Date format should actually be moved in the input placeholder like in contributions filters and advanced search.

And the translatable string should idealy be changed from `(yyyy-mm-dd format)` to `yyyy-mm-dd format` or even only `yyyy-mm-dd`
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/placeholder

Reply:
Well.. Using placeholder is a good idea, but when a date is already present in the field, the user cannot know what format is required.
There is a date picker on all date fields; yes, but some may prefer to work from keyboard only.

I wonder if the "example" span should not be left as is (we can still add placeholder, I'll work on translations once current PR has been merged).

Reply:
I've created draft PR #451 so I won't forget that :)

Reply:
> Don't you think the date already present is an _actual example_ of the required date format ? 🤔

It's a good point indeed... Maybe am I just too much used to this display? Let's go for placeholder only display, we can adjust quite simply in the future if required ;)